migrations/2022-08-09-142755_create_tables_weather_bot/up.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE chats (
   user_id BYTEA,
   state client_state DEFAULT 'initial' NOT NULL, -- Initial
   selected VARCHAR(80),
-  offset CHAR,
+  "offset" BYTEA,
```

Comment:
why does it have to be bytea? I think integer is fine?

Reply:
Cheap reason.